libafl/src/events/llmp.rs

```diff
@@ -892,6 +892,25 @@ where
     }
 }
 
+/// Specify if the State must be persistent over restarts
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub enum LlmpSaveState {
```

Comment:
Maybe `LlmpShouldSafeState` or `LlmpSaveStateOption` (because this enum isn't the actual state)